tests/playwright/src/runner/podman-desktop-runner.ts

```diff
@@ -233,26 +267,30 @@ export class PodmanDesktopRunner {
       rmSync(rawTracesPath, { recursive: true, force: true, maxRetries: 5 });
     }
 
+    let testStatus: string | undefined;
     try {
-      if (test.info().status === 'failed') return;
+      testStatus = test.info().status;
+      console.log(`Test finished with status:${testStatus}`);
     } catch (err) {
       console.log(`Caught exception in removing traces: ${err}`);
       return;
     }
 
-    if (!process.env.KEEP_TRACES_ON_PASS) {
-      const tracesPath = join(this._testOutput, 'traces', `${this._videoAndTraceName}_trace.zip`);
-      if (existsSync(tracesPath)) {
-        console.log(`Removing traces folder: ${tracesPath}`);
-        rmSync(tracesPath, { recursive: true, force: true, maxRetries: 5 });
+    if (testStatus && testStatus === 'passed') {
```

Comment:
This additional nested if statement can be removed, the issue can be addressed as per the suggestion from above.

Reply:
fixed